test/suitesOrder.js

```diff
@@ -0,0 +1,157 @@
+var execHistoryCopy,
+	execHistory = [];
+
+(function( suite, test, beforeEach, afterEach ) {
```

Comment:
Can we use these directly instead of aliasing? Our API should not warrant this. If we don't even want to use it ourselves, we shouldn't ship it. I haven't found it to be an issue though. But I'm happy to revisit it. In our own tests should serve as good example though. I'd rather not encourage people copy this.